### PR TITLE
[WIP] Style adjustments (Inspect & overall)

### PR DESCRIPTION
**Pull request by Westsaid**
image

### CODE REVIEW

File: code/ui/GameTimer.cs [hunk 1]
```diff
@@ -6,27 +6,46 @@ namespace TTTReborn.UI
...
+                TimeLabel = Add.Label("00:00", "timelabel");
             }
+            public override void Tick()
+            {
+                // TODO: Handle if Instance is null and if Round is null.
+                if (Game.Instance.Round is Rounds.WaitingRound)
+                {
+                    TextLabel.Text = $"{Game.Instance.Round.RoundName}...";
+                    TimeLabel.Text = "";
+                    TimeLabel.AddClass("waiting");
```
Comment (mzegar) [inline code/ui/GameTimer.cs:37]: We've used `hide` as a class name for hiding stuff, maybe we should just stick to that.

**Review by mzegar — COMMENTED** [review]
Minor comment, I like the changes to the UI though.

File: code/ui/GameTimer.cs (hunk 1 above)
Comment (Westsaid) [reply code/ui/GameTimer.cs:37]: Good point, solved! :)

**Comment by Westsaid** [comment]
I have another thing that really bugs me that I want to change for this PR. The colors in the InfoFeed are set to .Red & .Green. They are based on the colors given to the Role in e.g. the "TraitorRole.cs":
image

I've tried to change this to [Color.FromRgb(...)](https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.color.fromrgb?view=net-5.0) but this isn't available here...

Does someone of you have a clue?

File: code/ui/GameTimer.cs [hunk 2]
```diff
@@ -6,27 +6,49 @@ namespace TTTReborn.UI
 {
     public class GameTimer : Panel
     {
-        public Label TimeLabel { set; get; }
 
+        private GameTimerContent gameTimerContent;
```
Comment (Alf21) [inline code/ui/GameTimer.cs:10]:
```suggestion
        private GameTimerContent gameTimerContent;

```

File: code/ui/GameTimer.cs [hunk 3]
```diff
@@ -6,27 +6,49 @@ namespace TTTReborn.UI
...
+            public Label TextLabel { set; get; }
+            public Label TimeLabel { set; get; }
+
+            public GameTimerContent(Panel parent)
             {
-                TimeLabel.Text = $"{Game.Instance.Round.RoundName}...";
+                Parent = parent;
 
-                return;
+                TextLabel = Add.Label("asd", "textlabel");
```
Comment (Alf21) [inline code/ui/GameTimer.cs:27]:
```suggestion
                TextLabel = Add.Label("", "textlabel");
```

File: code/ui/GameTimer.cs [hunk 4]
```diff
@@ -6,27 +6,49 @@ namespace TTTReborn.UI
...
+                TimeLabel = Add.Label("00:00", "timelabel");
             }
+            public override void Tick()
+            {
+                // TODO: Handle if Instance is null and if Round is null.
+                if (Game.Instance.Round is Rounds.WaitingRound)
+                {
+                    TextLabel.Text = $"{Game.Instance.Round.RoundName}...";
+                    TimeLabel.AddClass("hide");
+                    TimeLabel.AddClass("waiting");
```
Comment (Alf21) [inline code/ui/GameTimer.cs:37]: You could store the bool `Game.Instance.Round is Rounds.WaitingRound` and then use `TimeLabel.SetClass("class", bool)` so reduce code

Comment (Alf21) [reply code/ui/GameTimer.cs:37]: Additionally, `TimeLabel` needs to be set to `""` here

File: code/ui/GameTimer.cs [hunk 5]
```diff
@@ -6,27 +6,49 @@ namespace TTTReborn.UI
...
+                {
+                    TimeLabel.RemoveClass("hide");
+                    TimeLabel.RemoveClass("waiting");
+                    TextLabel.Text = $"{Game.Instance.Round.RoundName}:";
+                    TimeLabel.Text = $"{Game.Instance.Round.TimeLeftFormatted}";
+                }
+
 
-            TimeLabel.Text = $"{Game.Instance.Round.RoundName}: {Game.Instance.Round.TimeLeftFormatted}";
+                // AddClass("playing");
```
Comment (Alf21) [inline code/ui/GameTimer.cs:48]: ?

File: code/ui/GameTimer.scss [hunk 6]
```diff
@@ -16,7 +16,21 @@ GameTimer {
...
     font-family: Poppins;
 
+    GameTimerContent {
+        display: flex;
+        justify-content: space-around;
+        width: 100%;
+        .timelabel {
+            margin-left: 10px;
+            width: 65px;
+            &.waiting{
```
Comment (Alf21) [inline code/ui/GameTimer.scss:26]:
```suggestion

        .timelabel {
            margin-left: 10px;
            width: 65px;

            &.waiting{
```

File: code/ui/InspectMenu.scss [hunk 7]
```diff
@@ -27,49 +27,56 @@ InspectMenu {
...
+                height: 75px;
+                width: 75px;
+                background-color: rgba(0, 0, 0, 0.7);
                 border: 1px solid #031421;
                 border-radius: 4px;
                 text-align: center;
                 // transition: all 0.05s linear;
-
+                margin-right: 5px;
                 &.playericon {
```
Comment (Alf21) [inline code/ui/InspectMenu.scss:75]:
```suggestion

                &.playericon {
```

File: code/ui/WeaponSelection.scss [hunk 8]
```diff
@@ -37,13 +33,25 @@ WeaponSelection {
...
+                background-color: #ff9800;
                 font-weight: 600;
             }
 
             &.ammolabel {
                 right: 10px;
                 position: absolute;
+                font-size: 20px;
+                padding-top: 2px;
+                &.active {
```
Comment (Alf21) [inline code/ui/WeaponSelection.scss:45]:
```suggestion

                &.active {
```

File: code/ui/WeaponSelection.scss [hunk 9]
```diff
@@ -37,13 +33,25 @@ WeaponSelection {
...
+                    font-size: 22px;
+                }
+            }
+        }
+
+        &.active {
+            background-color: rgba(0, 0, 0, 0.9);
+            Label {
```
Comment (Alf21) [inline code/ui/WeaponSelection.scss:53]:
```suggestion

            Label {
```

**Review by Alf21 — CHANGES_REQUESTED** [review]
Most of all, just code style

File: code/ui/GameTimer.cs (hunk 5 above)
Comment (Westsaid) [reply code/ui/GameTimer.cs:48]: Ups forgot to delete

File: code/ui/GameTimer.cs (hunk 4 above)
Comment (Westsaid) [reply code/ui/GameTimer.cs:37]: Good suggestion, I've implemented this!

**Comment by Westsaid** [comment]
@Alf21 Got trough your review and made those changes.

File: code/ui/GameTimer.cs [hunk 10]
```diff
@@ -6,27 +6,45 @@ namespace TTTReborn.UI
...
+                bool isWaitingRound = Game.Instance.Round is Rounds.WaitingRound;
+                if (isWaitingRound)
+                {
+                    TextLabel.Text = $"{Game.Instance.Round.RoundName}...";
+                }
+                else
+                {
+                    TextLabel.Text = $"{Game.Instance.Round.RoundName}:";
+                    TimeLabel.Text = $"{Game.Instance.Round.TimeLeftFormatted}";
+                }
```
Comment (Alf21) [inline code/ui/GameTimer.cs:42]:
```suggestion
                bool isWaitingRound = Game.Instance.Round is Rounds.WaitingRound;

                if (isWaitingRound)
                {
                    TextLabel.Text = $"{Game.Instance.Round.RoundName}...";
                    TimeLabel.Text = "";
                }
                else
                {
                    TextLabel.Text = $"{Game.Instance.Round.RoundName}:";
                    TimeLabel.Text = $"{Game.Instance.Round.TimeLeftFormatted}";
                }

```

File: code/ui/GameTimer.scss [hunk 11]
```diff
@@ -16,7 +16,23 @@ GameTimer {
...
+                display: none;
+            }
+        }
+    }
+
     Label {
         font-weight: bold;
     }
+    display: flex;
```
Comment (Alf21) [inline code/ui/GameTimer.scss:37]: Is this really needed? :o

Comment (Westsaid) [reply code/ui/GameTimer.scss:37]: Indeed not!

**Review by Alf21 — APPROVED** [review]

**Comment by Westsaid** [comment]
Updated Role Colors: 
image

**Pull request merged**